DivvyUp.sol

```diff
@@ -784,45 +722,28 @@ contract DivvyUp is ERC20Interface, Owned, DivvyUpInterface {
     /*==========================================
     =            INTERNAL FUNCTIONS            =
     ==========================================*/
-    function purchaseTokens(uint256 incomingCounter, address referredBy)
+    function purchaseTokens(uint256 incomingCounter, address)
```

Comment:
The address needs to be removed from this function. There's no address to pass in.